### PR TITLE
docs: announcement for 1.23

### LINKED ISSUE #181: Announce 1.23.0, 1.22.3 and 1.21.5

### PR DESCRIPTION
**Pull request by gbartolini**
Closes #181

### CODE REVIEW

**Comment by gbartolini** [comment]
DO NOT MERGE YET, just a stub

File: content/blog/cloudnative-pg-1-23.0-released/index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+The **CloudNativePG Community** is excited to announce the release of version
+1.23.0 of the **CloudNativePG Operator**!
+
+This release brings a host of new features and enhancements, including support
+for PostgreSQL image catalogs, synchronization of user-defined replication
+slots, and Pod Disruption Budget (PDB) configuration.
+
+We've revised our Community support policy to prioritize one minor release at a
+time, enhancing our focus and resources. Furthermore, we've extended the
+supplementary support period for the prior minor release to three months.
```
Comment (jsilvela) [inline content/blog/cloudnative-pg-1-23.0-released/index.md:29]: This makes it sound like we only have one release?
My understanding was we were going from 3 to 2?

File: content/blog/cloudnative-pg-1-23.0-released/index.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+## What's New in 1.23
+
+PostgreSQL Image Catalogs
+:   Say goodbye to PostgreSQL version management headaches! With image catalogs
+    based on major versions, managing your database fleet has never been easier.
+    Simply request the PostgreSQL major version you need and control how your
+    databases stay up-to-date. We've introduced two new resources
+    (`ClusterImageCatalog` and `ImageCatalog`) and a new stanza
+    (`spec.imageCatalogRef`), setting the stage for future defaults. You can use
+    the Community provided catalogs, third-party ones, or even build your own.
```
Comment (jsilvela) [inline content/blog/cloudnative-pg-1-23.0-released/index.md:40]: "future defaults" has a potential negative connotation.

```suggestion
    (`spec.imageCatalogRef`), setting the stage for simpler setting of default images.
    The Community will provide catalogs, and you could also use third-party ones,
    or even build your own.
```

File: content/blog/cloudnative-pg-1-23.0-released/index.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+Synchronization of User-Defined Replication Slots
+:   Ensure seamless failover with extended physical replication slot
+    synchronization, now covering user-defined slots persist after failover.
```
Comment (jsilvela) [inline content/blog/cloudnative-pg-1-23.0-released/index.md:44]:
```suggestion
    synchronization, now covering persistence of user-defined slots after failover.
```

**Review by jsilvela — APPROVED** [review]

File: content/blog/cloudnative-pg-1-23.0-released/index.md (hunk 1 above)
Comment (jsilvela) [inline content/blog/cloudnative-pg-1-23.0-released/index.md:29]:
```suggestion
We've revised our Community support to enhance our focus and optimize resources.
The supported versions will be the last minor, plus the previous minor, which will be
supported for three months once the new minor is out.
This is in effect adding two more months to the outgoing minor over what we used to have.
```

**Review by jsilvela — DISMISSED** [review]
The explanation on support policy was confusing me. Left a suggestion.

**Review by leonardoce — APPROVED** [review]

File: content/blog/cloudnative-pg-1-23.0-released/index.md (hunk 1 above)
Comment (gbartolini) [reply content/blog/cloudnative-pg-1-23.0-released/index.md:29]: Effectively, this is what is going to happen. We went from 2 minor releases + 1 additional month, to 1 minor release + 3 months. So for at least 3 months we need to support 2 minor releases. With a 3 month delivery time of a release (like now), it still brings to a total of roughly 6 months of support.

**Review by mnencia — APPROVED** [review]

**Pull request merged**